src/analyzer.ts

```diff
@@ -58,6 +58,11 @@ function isDependencyExternal(url: string) {
 /**
  * Get a longer, single-line error message for logging and exeption-handling
  * analysis Warning objects.
+ *
+ * Note: We cannot use WarningPrinter.printWarning() from the polymer-analyzer
+ * codebase because it's reported source ranges don't match the users source
```

Comment:
nits (because I'm a crazy person): 
  `it's` -> `its`
  `users` -> `user's` or `users'`